### ISSUE TITLE
Did Powerlevel 10k break my terminal?

### ISSUE THREAD

**Issue by timmyreal** [romkatv/powerlevel10k#2142]
I'm a little bit freaked out by this.

I kept getting an error message about Instant Prompt or whatever, so I did as it asked me to do. Nothing seemed to work, but when I put `typeset -g POWERLEVEL9K_INSTANT_PROMPT=quiet` into .zshrc and restarted iterm2, it started causing problems. It doesn't recognize even basic commands like `ls` or `vim`. I can't even use the `p10k configure` command because it says I don't have permission. I changed some things around in the configuration file again, but things aren't going back to normal. Help!

Here's a screenshot:
<img width="660" alt="Screen Shot 2023-01-04 at 11 37 58 AM" src="https://user-images.githubusercontent.com/82277734/210604465-345e10d6-e377-4476-9d5f-4e250d1221ed.png">

Here's what the top of configuration file looks like now:
<img width="425" alt="Screen Shot 2023-01-04 at 11 38 30 AM" src="https://user-images.githubusercontent.com/82277734/210604561-15d9a37c-2073-469f-8af9-8e603e01a383.png">

**Comment by romkatv** [comment]
Your zshrc has multiple errors. You need to fix them. The first error is visible on the screenshot: the name of the file is misspelled (there is a stray `#` in it). The second error is on line 121. I don't know what that line is.

Whenever you expect that powerlevel10k is causing problems, try disabling it. If the errors don't go away, they aren't related to powerlevel10k.

**Comment by timmyreal** [comment]
Ok, I fixed those errors, restarted iterm, and it still didn't fix things. I then disabled every single reference to powerlevel10k in .zshrc, restarted iterm, but it still isn't fixed. I understand that technically it means it's not a problem with powershell10k, but also everything was working fine until I added this one line of code recommended by the technical documentation.

Is there anything in your experience that would make a terminal completely stop recognizing even basic Unix commands?

Here's the output now.
<img width="483" alt="Screen Shot 2023-01-04 at 11 51 44 AM" src="https://user-images.githubusercontent.com/82277734/210607271-01478f2d-8878-4c4f-8890-69b07559c33d.png">
